### PR TITLE
Coordinate and padding clarifications

### PR DESCRIPTION
**Pull request by lilleyse**
@pjcozzi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,15 +198,15 @@ An optional `transform` property (not shown above) defines a 4x4 affine transfor
 
 ### Coordinate System and Units
 
-3D Tiles use a right-handed Cartesian coordinate system, that is, the cross product of x and y yields z. 3D Tiles define the z axis as up for local Cartesian coordinate systems (see the [Tile transform](#tile-transform) section).  A tileset's global coordinate system will often be [WGS84 coordinates](http://earth-info.nga.mil/GandG/publications/tr8350.2/wgs84fin.pdf), but it doesn't have to be, e.g., a power plant may be defined fully in its local coordinate system for using with a modeling tool without a geospatial context.
+3D Tiles use a right-handed Cartesian coordinate system, that is, the cross product of x and y yields z. 3D Tiles define the z axis as up for local Cartesian coordinate systems (see the [Tile transform](#tile-transform) section).  A tileset's global coordinate system will often be [ECEF](https://en.wikipedia.org/wiki/ECEF) coordinates, but it doesn't have to be, e.g., a power plant may be defined fully in its local coordinate system for using with a modeling tool without a geospatial context.
```
Comment (pjcozzi) [inline README.md]: Throughout...

In general, do not link to Wikipeda as the seminal reference.

Also, it is not clear to me why this is now ECEF...I thought we were making EPSG:4326 WGS84 explicit...less readers are going to know what ECEF is without a bit more of a description.

File: TileFormats/BatchTable/README.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ A Batch Table is composed of two parts: a JSON header and an optional binary bod
 
 ![batch table layout](figures/batch-table-layout.png)
 
+See [Implementation Notes](#implementation-notes) for details about padding between the json and binary sections.
```
Comment (pjcozzi) [inline TileFormats/BatchTable/README.md]: json -> JSON.

Should the section just be up here?  Should this be an implementation note?  Is it normative, i.e., part of the actual spec?

File: schema/tile.schema.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
         },
         "transform" : {
             "type" : "array",
-            "description" : "A floating-point 4x4 affine transformation matrix, stored in column-major order, that transforms the tile's content, i.e., its features and content.boundingVolume, and boundingVolume and viewerRequestVolume from the tile's local coordinate system to the parent tile's coordinate system, or tileset's coordinate system in the case of the root tile.  transform does not apply to geometricError nor does it apply any volume property when the volume is a region, which is defined in WGS84.",
+            "description" : "A floating-point 4x4 affine transformation matrix, stored in column-major order, that transforms the tile's content, i.e., its features and content.boundingVolume, and boundingVolume and viewerRequestVolume from the tile's local coordinate system to the parent tile's coordinate system, or tileset's coordinate system in the case of the root tile.  transform does not apply to geometricError nor does it apply any volume property when the volume is a region, which is defined in EPSG:4326 coordinates.",
```
Comment (pjcozzi) [inline schema/tile.schema.json]: Throughout say "WGS84 / EPSG:4326"

The confusion is that WGS84 defines both the ellipsoid radii and the Cartesian coordinate system.  Here we mean the coordinate system.

**Comment by lilleyse** [comment]
Updated.

> Should the section just be up here? Should this be an implementation note? Is it normative, i.e., part of the actual spec?

Yeah.. I definitely see the benefit of making padding rules part of the spec, not just an implementation note. I'll open a separate PR for that.

**Comment by pjcozzi** [comment]
Submitted #225.

Looks good!

**Pull request merged**